engine/src/main/java/org/camunda/bpm/engine/impl/TaskQueryImpl.java

```diff
@@ -50,6 +50,43 @@
  */
 public class TaskQueryImpl extends AbstractQuery<TaskQuery, Task> implements TaskQuery {
 
+  /**
```

Comment:
Right now this is Javadoc for the field `serialVersionUID`.
Was this supposed to be 
```suggestion
  /*
```
?

Reply:
No, it was just a mistake while typing. I committed the suggestion. Thanks!